packages/dds/tree/src/simple-tree/core/unhydratedFlexTree.ts

```diff
@@ -322,7 +322,7 @@ const unparentedLocation: LocationInField = {
 	index: -1,
 };
 
-class UnhydratedFlexTreeField implements FlexTreeField {
+abstract class UnhydratedFlexTreeField implements FlexTreeField {
```

Comment:
Changes to this file were just general cleanup, ensuring future adding of field kinds errors if more work to supported them is needed here instead of silently mostly working.